src/helm.exec.ts

```diff
@@ -263,7 +275,7 @@ export function helmGet(resourceNode?: ClusterExplorerNode) {
     if (!resourceNode) {
         return;
     }
-    if (resourceNode.nodeType !== NODE_TYPES.helm.release) {
+    if ((resourceNode.nodeType !== NODE_TYPES.helm.release || NODE_TYPES.helm.history)) {
```

Comment:
There is an issue here. It's interpreted like  (resourceNode.nodeType !== NODE_TYPES.helm.release || true) as NODE_TYPES.helm.history has a value. The helm history treeItem command is never executed bc it gets in this if.

Reply:
Fixed this, thanks for catching this